### DISCUSSION TITLE
USB Token Redirection - A3 Cardified

### DISCUSSION THREAD

**Discussion by MarcosAG02** [neutrinolabs/xrdp#2669] [Q&A]
Good afternoon people!

I have a Xubuntu environment at work, where developers using private Windows workstations connect through a VPN and Windows RDP.
When configuring RDP on these particular computers to redirect USB drives and ports, these drives appear in the "Thinclient_drives" folder, but the USB tokens (with A3 certificate) do not appear, nor are they recognized by the authentication software (Safenet authentication client).

Is this USB Token redirection possible from Windows to Linux?

Is there any configuration I need to do to make this redirection work?

I thank the help of all!

**Comment by matt335672** [comment]
Hi @MarcosAG02 

At the moment, no it isn't possible.

The RDP protocol would support these by USB direction, or possibly by smartcard redirection. The first of these hasn't been attempted, and the second needs a lot of work to get to a working state.

**Reply by MarcosAG02** [reply]
@matt335672 
Thanks for the answer.
From what I researched, there are some paid solutions that claim to do this redirection.

I'll look and test some.
